lib/services/chat_service.dart

```diff
@@ -28,6 +31,30 @@ class ChatService {
   Stream<ChatListTileDataModel> get chatListStream => _chatListStream;
   Stream<ChatMessage> get chatMessagesStream => _chatMessagesStream;
 
+  // Stream<QueryResult> getMessagesFromDirectChat() async* {
```

Comment:
Should comments be removed before merging?

Reply:
I want to let this stay. I'll be using this code for subscriptions.